assets/js/blocks/cart-checkout/shared/use-view-switcher.tsx

```diff
@@ -36,7 +36,7 @@ export const useViewSwitcher = (
 				}
 				controls={ views.map( ( view ) => ( {
 					...view,
-					title: view.label,
+					title: <span>{ view.label }</span>,
```

Comment:
Added a span to match Gutenberg and to allow more specific XPath selectors.

Reply:
We might consider switching to `MenuItem` instead of `controls` later.